mne/epochs.py

```diff
@@ -409,7 +409,6 @@ def __init__(self, info, data, events, event_id=None, tmin=-0.2, tmax=0.5,
             # ensure metadata matches original events size
             self.selection = np.arange(len(events))
             self.events = events
-            self.metadata = metadata
```

Comment:
The intent was to remove this here, because it is set again  in line 446 ... and in the intermediate lines, `self.metadata` does not appear to be needed.

Removing this call here would fix the duplicate log issue. But locally, this turns into an issue for existing tests.

Reply:
> The intent was to remove this here, because it is set again in line 446

Not really – line 446 is the `else` branch of this `if` block. Am I missing something?

Reply:
nope 446 is on the same indentation level. You might have confused that with the third call of setting metadata at line 462 --> that one is indeed in the `else` branch of the `if` block. Unless I am completely blind :-D